node/nanoexpress/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  github: dalisoft/nanoexpress
```

Comment:
perhaps better to redirect to https://nanoexpress.js.org/ ? using `website: nanoexpress.js.org` :stuck_out_tongue: 
this will create a link in `README` to https://nanoexpress.js.org/

Reply:
I did it, now looks good?